content/docs/practices/naming.md

```diff
@@ -26,7 +26,7 @@ A metric name...
    (exported by many client libraries)
  * <code><b>http</b>\_request\_duration\_seconds</code>
    (for all HTTP requests)
-* ...must have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
+* ...must have a single unit (i.e. do not mix seconds with milliseconds, or bits with bytes).
```

Comment:
"have" is perhaps the wrong word here anyway. A metric has a unit, but a metric name contains a unit or refers to it…

To also address the quantity aspect:

```suggestion
* ...must refer to a single unit (e.g. do not mix seconds with milliseconds) and to a single quantity (e.g. do not mix request size with request duration).
```

This is kind of related or even redundant with L30 and L40, but perhaps being overly explicit is not a bad thing here.